pkg/kubelet/kubelet.go

```diff
@@ -2846,13 +2853,15 @@ func (kl *Kubelet) handlePodResourcesResize(pod *v1.Pod) *v1.Pod {
 		// Update pod resource allocation checkpoint
 		if err := kl.statusManager.SetPodAllocation(updatedPod); err != nil {
 			//TODO(vinaykul,InPlacePodVerticalScaling): Can we recover from this in some way? Investigate
+			kl.recorder.Eventf(updatedPod, v1.EventTypeWarning, events.ResizeFailed, "Resize pod %s failed: %v", klog.KObj(updatedPod), err)
```

Comment:
I don't think we should emit an event for this, it's already captured by the logs. I think we mainly want to emit events at the state transitions (i.e. when ResizeStatus changes)

Reply:
removed